client/src/featureform/dashboard_metadata.py

```diff
@@ -62,6 +62,11 @@ def deliver_static(asset):
     return dashboard_app.send_static_file("static/" + asset)
 
 
+@dashboard_app.route("/data/version")
+def version():
+    return {"version": get_package_version()}
```

Comment:
Is this the same as the client version? do we want to know the cluster version?

Reply:
IIRC we're using the python package version in local mode.